data/mods/Aftershock/mobs/robots.json

```
@@ -279,23 +279,23 @@
     "id": "mon_hazmatbot",
     "type": "MONSTER",
     "copy-from": "mon_utilibot",
-    "name": "hazmat bot",
-    "description": "A utility robot designed for cleaning up waste material in hazardous conditions."
+    "name": "TsKBEM Likvidator",
+    "description": "The Likvidator was designed by the TsKBEM design bureau for the Soviet Union as an answer to liquidators post-Chernobyl.  Rather than using humans, easily suspectible to radiation sickness and hard to repair, the Likvidator would do the duties of taking care of toxic hazardous waste instead.  Likvidators are roughly human-sized, bipedal bulky machines; crafted for a utilitarian purpose of cleaning up toxic waste rather than aesthetic."
   },
   {
     "id": "mon_utilibot_butler",
     "type": "MONSTER",
     "copy-from": "mon_utilibot",
-    "name": "butler-bot",
+    "name": { "str": "ENIAC Famulus", "str_pl": "ENIAC Famuli" },
     "color": "dark_gray",
-    "description": "A luxury model utility robot for domestic use."
+    "description": "Top of the line and luxury built, the Famulus is a humaniform robot, derived from the humaniform schematics by Wraitheon with slight alterations to give it a more human appearance.  It is designed to be a robotic servant, its AI tailored for the needs for each customer; and arguably one of the few commercial robots to pass the turing test."
```

```suggestion
    "description": "A top of the line luxury model, the Famulus is a humaniform robot, derived Wraitheon  schematics with slight alterations to give it a more human appearance.  Designed to be a robotic servant, its AI tailored for the needs for each customer, it is arguably one of the few commercial robots to pass the Turing test."
```
Not sure on the phrasing of the last part.